src/Sylius/Behat/Context/Api/Admin/ManagingTaxCategoriesContext.php

```diff
@@ -201,13 +221,37 @@ public function taxCategoryWithNamedElementShouldNotBeAdded(string $element, str
      */
     public function iShouldSeeSingleTaxCategoryInTheList(): void
     {
-        Assert::same($this->client->countCollectionItems(), 1);
+        Assert::same($this->responseChecker->countCollectionItems($this->client->getLastResponse()), 1);
+    }
+
+    /**
+     * @Then I should be notified that it has been successfully created
+     */
+    public function iShouldBeNotifiedThatItHasBeenSuccessfullyCreated(): void
+    {
+        Assert::true($this->responseChecker->isCreationSuccessful($this->client->getLastResponse()));
```

Comment:
There are a few `true/false` assertion without any comment, which will end up with an awesome error: "Expected true, got false"

We should do something about it in the next iteration